src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import classes from './VideoCard.module.css';
+
+const VideoCard = (props) => {
+  return (
+    <div className={classes['video-card']} data-testid="VideoCard">
+      <img
+        className={classes['video-card-image']}
+        src={props.videoData.thumbnail}
+      />
```

Comment:
you have a warning on the console, remember to add the alt prop in case the image can't be loaded
```
<img
        className={classes['video-card-image']}
        src={props.videoData.thumbnail}
        alt={ title }
 />
```